openapi/atlas-api-transformed.yaml

```
@@ -21769,6 +21769,7 @@ components:
           example: TOO_MANY_GROUP_NOTIFICATIONS
         parameters:
           type: array
+          description: Parameter uses to give more information about the error.
```

👀  I know this is a thing on the spec but I guess this should be 
```suggestion
          description: Parameter used to give more information about the error.
```

---

Created jira to fix upstream